test/test-integration/src/test/java/com/alipay/sofa/rpc/test/triple/TripleServerTest.java

```diff
@@ -168,8 +163,6 @@ public void testExposeTwice() {
             .setRef(new GreeterImpl())
             .setServer(serverConfig);
 
-        providerConfig.setParameter(TripleConstant.TRIPLE_EXPOSE_OLD, "true");
-
         try {
             providerConfig.export();
             Assert.fail();
```

Comment:
Now we support expose service with multi uniqueId, so it will not fail. By the way , when we expose multi uniqueId. when we unexpose one ,the other should still work.

Reply:
这个做了下修改，已经支持了